test/CMakeLists.txt

```diff
@@ -72,6 +72,20 @@ endif()
 compile_benchmark_test(complexity_test)
 add_test(complexity_benchmark complexity_test --benchmark_min_time=0.01)
 
+if(UNIX)
+    find_path(PAPI_PREFIX NAMES include/papi.h)
+    find_path(PAPI_INCLUDE_DIRS NAMES papi.h HINTS ${PAPI_PREFIX}/include)
+    find_library(PAPI_LIBRARIES NAMES libpapi.a papi HINTS ${PAPI_PREFIX}/lib)
+    if(PAPI_INCLUDE_DIRS AND PAPI_LIBRARIES)
+        find_library(LIBPFM pfm) # needed in Ubuntu 16.04
+        include_directories(${PAPI_INCLUDE_DIRS})
+        compile_benchmark_test(papi_test)
+        add_test(papi_test papi_test --benchmark_min_time=0.01)
+        target_link_libraries(papi_test ${PAPI_LIBRARIES} ${LIBPFM})
+    endif()
+endif(UNIX)
+
+
```

Comment:
D'oh, this was also accidentally put in. BTW, I take this opportunity to ask: would you be interested in an user-defined counter example with PAPI counters? If so, I'll make a separate PR with this. Otherwise I'll just remove this.